servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/SubscriptionDescription.java

```diff
@@ -137,14 +134,13 @@ public Duration getDefaultMessageTimeToLive() {
      * See {@link #getDefaultMessageTimeToLive()}
      */
     public void setDefaultMessageTimeToLive(Duration defaultMessageTimeToLive) {
-        if (defaultMessageTimeToLive != null &&
-                (defaultMessageTimeToLive.compareTo(ManagementClientConstants.MIN_ALLOWED_TTL) < 0 ||
-                        defaultMessageTimeToLive.compareTo(ManagementClientConstants.MAX_ALLOWED_TTL) > 0))
-        {
+        if (defaultMessageTimeToLive != null
```

Comment:
>f (defaultMessageTimeToLive != null [](start = 9, length = 35)

May be you can change it to defaultMessageTimeToLive == null. That way it will throw a validation exception for the null case too. Can you do for every Description class where this null check is not done?

Reply:
Do you want to have all null checking for all setter methods that pass Object as parameters in the Description class? Such as     
public void setAuthorizationRules(List<AuthorizationRule> authorizationRules) {
        this.authorizationRules = authorizationRules;
    }